examples/MNIST/resizenorm.yaml

```diff
@@ -5,6 +5,12 @@ definitions:
   associate_spec:
     name: associate_spec
     primitive: List[str]
+  flatten.inputs.array:
```

Comment:
Do we even need these files anymore?

Reply:
Oh! Since we are using diagrams now, we don't need them. I'll remove them!